src/fillTank.test.js

```diff
@@ -1,11 +1,147 @@
 'use strict';
 
 describe('fillTank', () => {
-  // const { fillTank } = require('./fillTank');
+  const { fillTank } = require('./fillTank');
 
   it('should ', () => {
 
   });
 
   // write tests here
+  describe('fillTank', () => {
+    it('should be a function', () => {
+      expect(typeof fillTank).toBe('function');
+    });
+  });
+
+  it('should work properly', () => {
+    const customer = {
+      money: 60,
+      vehicle: {
+        maxTankCapacity: 50,
+        fuelRemains: 10,
+      },
+    };
+    const fuelPrice = 2;
+
+    fillTank(customer, fuelPrice, Infinity);
+    expect(customer.vehicle.fuelRemains).toBe(50);
+    expect(customer.money).toBe(0);
+  });
+
+  it('should fill tank if amount is not present', () => {
+    const customer = {
+      money: 300,
+      vehicle: {
+        maxTankCapacity: 30,
+        fuelRemains: 0,
+      },
+    };
+    const fuelPrice = 20;
+
+    fillTank(customer, fuelPrice);
+    expect(customer.vehicle.fuelRemains).toBe(30);
+  });
+
+  it('should fill tank with amount value', () => {
+    const customer = {
+      money: 1000,
+      vehicle: {
+        maxTankCapacity: 30,
+        fuelRemains: 10,
+      },
+    };
+    const fuelPrice = 1;
+
+    fillTank(customer, fuelPrice, 10);
+    expect(customer.vehicle.fuelRemains).toBe(20);
+  });
+
+  it('should not fill more than customer can pay', () => {
+    const customer = {
+      money: 100,
+      vehicle: {
+        maxTankCapacity: 500,
+        fuelRemains: 10,
+      },
+    };
+    const fuelPrice = 5;
+
+    fillTank(customer, fuelPrice, Infinity);
+    expect(customer.vehicle.fuelRemains).toBe(30);
+    expect(customer.money).toBe(0);
+  });
+
+  it('should discarde number of fuelRemains to 0,0', () => {
+    const customer = {
+      money: 100,
+      vehicle: {
+        maxTankCapacity: 50,
+        fuelRemains: 10,
+      },
+    };
+    const fuelPrice = 6;
+
+    fillTank(customer, fuelPrice, Infinity);
+    expect(customer.vehicle.fuelRemains).toBe(26, 6);
```

Comment:
The expectation `expect(customer.vehicle.fuelRemains).toBe(26, 6);` seems incorrect. The `toBe` matcher expects a single argument. If you intend to check for a specific value, ensure that the expected value is correct and singular.